src/panels/config/integrations/ha-integration-header.ts

```diff
@@ -8,6 +16,13 @@ import { domainToName, IntegrationManifest } from "../../../data/integration";
 import { HomeAssistant } from "../../../types";
 import { brandsUrl } from "../../../util/brands-url";
 
+const QUALITY_SCALE_ICONS = {
+  silver: mdiHomeFloor2,
+  gold: mdiHomeFloor1,
```

Comment:
No too sure about these 2 icons, but did not find any other that would be better :man_shrugging:

Reply:
Shouldn't we only show this badge when it is either platinum or, if it isn't?

Reply:
Don't know :shrug: